Dockerfile

```diff
@@ -1,13 +1,10 @@
-FROM node:carbon-alpine
+FROM node:lts-alpine
 
 MAINTAINER Apiary <sre@apiary.io>
 
 ARG DREDD_VERSION=latest
 RUN echo ${DREDD_VERSION}
```

Comment:
Maybe worth putting this in same line as below `echo .. &&` so that it doesn't cause a separate layer in the docker image cache.

Although, `npm install dredd@<version>` prints the version so maybe it can just be omitted here?